machine-setup/README.md

```diff
@@ -0,0 +1,11 @@
+Steps to run the setupRancher.py and teardownRancher.py to setup and teardown machines for longhorn tests
+
+# first create a properties file to retrieve machine build results from container
+# it will contain CATTLE_TEST_URL and LONGHORN_BACKUP_SERVER_IP
+touch $PROPERTY_FILE_NAME
+
+# call setupRancher.py using environment variables to setup longhorn test machines
+docker run --rm -v $(pwd)/$PROPERTY_FILE_NAME:/$PROPERTY_FILE_NAME -e PROPERTY_FILE_NAME="$PROPERTY_FILE_NAME" -e GCE_SERVICE_KEY_JSON="$GCE_SERVICE_KEY_JSON" -e GCE_RANCHER_PROJECT_NAME="$GCE_RANCHER_PROJECT_NAME" -e GCE_RANCHER_PROJECT_ZONE="$GCE_RANCHER_PROJECT_ZONE" -e GCE_NFS_SERVER_NAME="$GCE_NFS_SERVER_NAME" -e GCE_RANCHER_SERVER_NAME="$GCE_RANCHER_SERVER_NAME" -e PACKET_HOST_NAMES="$PACKET_HOST_NAMES" -e PACKET_RANCHER_AUTH_TOKEN="$PACKET_RANCHER_AUTH_TOKEN" -e PACKET_RANCHER_PROJECT_ID="$PACKET_RANCHER_PROJECT_ID" -e PACKET_HOST_MACHINE_TYPE="$PACKET_HOST_MACHINE_TYPE" -e PACKET_HOST_OS_IMG="$PACKET_HOST_OS_IMG" -e PACKET_HOST_CLOUD_CONFIG="$PACKET_HOST_CLOUD_CONFIG" -e GCE_RANCHER_MACHINE_TYPE="$GCE_RANCHER_MACHINE_TYPE" -e GCE_RANCHER_OS_IMG="$GCE_RANCHER_OS_IMG" -e GCE_STARTUP_SCRIPT_RANCHER="$GCE_STARTUP_SCRIPT_RANCHER" -e GCE_STARTUP_SCRIPT_NFS="$GCE_STARTUP_SCRIPT_NFS" jimeng/longhorn-test:001 setupRancher.py
```

Comment:
1. Format. Put `-e` in each line otherwise it's hard to read.
2. At least have a example for what each parameter means. I don't think anyone can understand only by the name here...